### PR TITLE
getting-started / JAAS review

### LINKED ISSUE #1977: Review needed: getting-started

### PR DESCRIPTION
**Pull request by pmatulis**
Fixes #1977 

Notes:
 - Buttons sometimes use monofont and sometimes single-quoted. What is the standard for Juju docs? MAAS uses single-quotes.
 - In the JAAS introduction, the 3 compatible clouds link to their respective Juju pages. This may be a source of confusion. Users may follow these links and get lost. Initially, I expected these to be links to upstream (e.g. https://aws.amazon.com).
 - As noted on IRC, this getting-started page and the jaas-cli page duplicate information.

### CODE REVIEW

File: src/en/getting-started.md [hunk 1]
```diff
@@ -68,19 +72,18 @@ Press the green 'Start building' button to get started...
 
 </table>
 
-
 You build your [model][models] by adding and relating applications from the
-[Charm Store][charmstore]. Press the green '+' symbol in the middle of the UI
-to start finding applications in the Charm Store.
+[Charm Store][charmstore]. Press the green '+' symbol in the middle of the web
+UI to search for applications in the Store.
```
Comment (evilnick) [inline src/en/getting-started.md]: I would leave 'Charm Store'. We seem to have so many stores!

File: src/en/getting-started.md [hunk 2]
```diff
@@ -195,13 +199,12 @@ details  {
    Juju via a snap package. See the [install][install] for more information.
 
 
-^# From the Ubuntu PPA
+^# For Ubuntu
```
Comment (evilnick) [inline src/en/getting-started.md]: I think i prefer it as it was - this makes it seem like this is the part for Ubuntu users, when in reality the approved/default method is installing the snap

File: src/en/getting-started.md (hunk 1 above)
Comment (pmatulis) [inline src/en/getting-started.md]: OK

File: src/en/getting-started.md (hunk 2 above)
Comment (pmatulis) [inline src/en/getting-started.md]: OK

**Pull request merged**